client/my-sites/migrate/helpers/index.js

```diff
@@ -34,22 +34,6 @@ export function getImportSectionLocation( siteSlug, isJetpack = false ) {
 		? `https://${ siteSlug }/wp-admin/import.php`
 		: `/import/${ siteSlug }/?engine=wordpress`;
 }
-// Flow mapping dictionary, key is the path segment, value is the flow name
-const flowMapping = {
-	'import-focused': 'import-focused',
-};
-
-export function getImportFlowByURL() {
-	const url = window.location.href;
-	const parsedUrl = new URL( url );
-	const pathSegments = parsedUrl.pathname.split( '/' );
-	// E.g. setup/import-focused/import returns import-focused
-	if ( pathSegments.length >= 3 && pathSegments[ 2 ] in flowMapping ) {
-		return flowMapping[ pathSegments[ 2 ] ];
-	}
-
-	return 'onboarding-flow';
-}
```

Comment:
The `getImportFlowByURL` would basically:

- if there's a hit in the `flowMapping` object, remove the part of the URL associated with a step, and return the slug of the flow;
- otherwise return `'onboarding-flow`

Having removed the `import-focused` flow, the whole function is no longer necessary.